README.md

```diff
@@ -63,7 +63,8 @@ sudo pacman -S php php-fpm php-apcu nginx git
 Clone LibreY:
 
 ```sh
-mkdir -p /var/www/html
+sudo mkdir -p /var/www/html/LibreY
```

Comment:
not really sure using sudo here is the best idea, doesnt really address systems where doas is used: assuming that all these commands should be run as root would be much better.